### ISSUE TITLE
Add reaction rates to profile/history output

### ISSUE THREAD

**Issue by rjfarmer** [MESAHub/mesa#359]
We do keep getting questions about getting eps_nuc out for a certain rate, so we should just go and add these as possible outputs in profile/history data so people don't have to keep working it out themselves.

**Comment by fxt44** [comment]
i agree requests for this kind of output keeps coming up. as you replied, a run_star_extras for the 2019 summer school was the last time i worked on it. maybe a cleaner up-to-date version could live in contrib or be part of the standard mesa options for output?

**Comment by rjfarmer** [comment]
Yeah I was thinking we just hook it into the normal output. We can add the raw_rate, screened_rate, eps_nuc, and the reaction neu's.

**Comment by earlbellinger** [comment]
What are the units of these quantities? If I had to guess, it would be reactions/sec for raw_rate and screened_rate, erg/g/s for eps_nuc, and MeV for neuQ. (Unless something else was meant by the reaction neus?) 

Do we want neuQ in the history/profile files? The info can be retrieved from `show_net_reactions_info` and doesn't change during the run as far as I know, though I suppose they could be in the history and/or profile headers. 

How do we want this to be listed in the history/profile files? Perhaps (to take eps_nuc as an example) the user could specify either `eps_nuc` to get the full net, or `eps_nuc reaction_name` to get a specific reaction?

**Comment by rjfarmer** [comment]
> What are the units of these quantities? If I had to guess, it would be reactions/sec for raw_rate and screened_rate, erg/g/s for eps_nuc, and MeV for neuQ. (Unless something else was meant by the reaction neus?)

Yes on raw_rate, screened_rate, and eps_nuc. For the reactions neu's I meant  screened_rate * neuQ so that would also be erg/g/s (the rate of energy lost to reaction neutrinos in each reaction) 

> Do we want neuQ in the history/profile files? The info can be retrieved from `show_net_reactions_info` and doesn't change during the run as far as I know, though I suppose they could be in the history and/or profile headers.

I don't think its really needed.

> How do we want this to be listed in the history/profile files? Perhaps (to take eps_nuc as an example) the user could specify either `eps_nuc` to get the full net, or `eps_nuc reaction_name` to get a specific reaction?

I would follow the style of the other multi-choice options. So it would be ``add_eps_nuc`` to get the full net and then ``eps_nuc reaction_name`` for each reaction.

**Comment by earlbellinger** [comment]
As of https://github.com/MESAHub/mesa/pull/392 we now have raw rates. We still need the screened rates as well as the eps_nuc and eps_neu from each rate.
